code/particle.py

```diff
@@ -0,0 +1,33 @@
+import numpy as np
+
+class Particle:
+    def __init__(self, nbr_variables, x_max, x_min):
+        self.nVariables = nbr_variables
```

Comment:
Use PEP8 naming convention --> self.nbr_variables = nbr_variables
You could also have a leading underscore in the name if you want, to signal that people should not mess with it after initialization.